pkgs/desktops/gnome/default.nix

```diff
@@ -277,6 +277,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   pomodoro = callPackage ./misc/pomodoro { };
 
+  solanum = callPackage ./misc/solanum { };
```

Comment:
New non-core apps should go to all-packages.nix.

Reply:
I saw pomodoro is here so I followed it. There is already a solanum, should I rename this one as gnome-solanum?

Reply:
Sounds alright

Reply:
Done!